specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AzureDevOpsConnectorListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "AzureDevOpsConnector_ListBySubscription": {
+            "$ref": "./examples/AzureDevOpsConnectorListBySubscription.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.SecurityDevOps/operations": {
+      "get": {
+        "description": "Returns list of operations supported by the Microsoft.SecurityDevOps provider.",
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Operation details.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/OperationsList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActionableRemediation": {
+      "type": "object",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/ActionableRemediationState"
+        },
+        "severityLevels": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "categories": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RuleCategory"
+          },
+          "x-ms-identifiers": []
+        },
+        "branchConfiguration": {
+          "$ref": "#/definitions/TargetBranchConfiguration"
+        }
+      }
+    },
+    "ActionableRemediationState": {
```

Comment:
>ActionableRemediationState [](http://example.com/codeflow?start=5&length=26)

Documentation to clarify - What does it even mean that an ActionableRemediation has a state of 'None', 'Disabled', or 'Enabled'?

Reply:
None implies it was never set, where as Enabled and Disabled imply overrides were set at repo, project or org level.

Reply:
A better description would be good.

Reply:
agreed, but here is why we will tackle this in next API version.
1. given our product is in code freeze and we generate swagger.
2. Additionally we need 7-10 days to test all swagger changes in Prod
3. We are super close to Microsoft Ignite, it is in less than 7 days.

Happy to take fix in next API version.